cytominer_eval/tests/test_operations/test_grit.py

```diff
@@ -11,17 +11,16 @@
 
 from cytominer_eval.operations import grit
 from cytominer_eval.transform import metric_melt
-from cytominer_eval.transform.util import (
-    assert_melt,
-    set_pair_ids,
+
+from cytominer_eval.utils.transform_utils import assert_melt, set_pair_ids
+from cytominer_eval.utils.operation_utils import assign_replicates
+from cytominer_eval.utils.grit_utils import (
     set_grit_column_info,
-)
-from cytominer_eval.operations.util import (
-    assign_replicates,
     get_grit_entry,
     calculate_grit,
 )
 
+
```

Comment:
This is something we need to change for some other files, I will point out below where we are getting the two spacing incorrect. This line is correct, but other files are not

Reply:
nice! Weird that black didn't catch the other one. I wonder if it's a version thing.

These version differences could get annoying... I wondered if there is a way to automate `black`, and it looks like there is via github actions! https://github.com/psf/black/actions/runs/17913292/workflow

I added #57 for us to track it there.

Thanks for adding the extra changes